### PR TITLE
Add host ports to task environment

### PR DESCRIPTION
**Pull request by marcusolsson**
Adds the assigned host ports to the environment to allow tasks to announce where to find them.

### CODE REVIEW

File: scheduler/task.go [hunk 1]
```diff
@@ -17,11 +19,15 @@ func createTaskInfo(task eremetic.Task, offer *mesos.Offer) (eremetic.Task, *mes
 
 	portMapping, portResources := buildPorts(task, offer)
 
+	env := &mesos.Environment{
+		Variables: buildEnvironment(task, portMapping),
```
Comment (keis) [inline scheduler/task.go]: Would make more sense for buildEnvironment to return a mesos.Environment maybe?

File: scheduler/task.go [hunk 2]
```diff
@@ -55,6 +61,18 @@ func buildEnvironment(task eremetic.Task) []*mesos.Environment_Variable {
...
 		})
 	}
+	for i, m := range portMappings {
+		environment = append(environment, &mesos.Environment_Variable{
+			Name:  proto.String(fmt.Sprintf("PORT%d", i)),
+			Value: proto.String(fmt.Sprintf("%d", *m.HostPort)),
+		})
+	}
+	if len(portMappings) > 0 {
+		environment = append(environment, &mesos.Environment_Variable{
```
Comment (keis) [inline scheduler/task.go]: Lets add some tests covering this case

**Comment by alde** [comment]
[LGTM](https://lgtm.in/i/owU0hvC3q)

**Comment by atl893** [comment]
How close is this to being merged, it's holding me up from getting something working, and I am willing to merge and build it myself, but don't want to take the time if it's gonna be in master soon. 

As an aside, it'd be nice if there was a way to pass mesos executor variables through to the docker host, as it'd save me some hackery. I might take a look at it if I get some free time this week.

**Comment by keis** [comment]
Looks ready to merge to me.

@atl893 Could you open a separate issue detailing what extra functionality you want exposed? PRs of course welcome but I figure it would be good to discuss it first

**Pull request merged**